lightly_studio/src/lightly_studio/api/routes/api/frame.py

```diff
@@ -72,6 +74,27 @@ def get_all_frames(
     )
 
 
+@frame_router.get("/bounds", response_model=Optional[VideoFrameFieldsBoundsView])
```

Comment:
Minor: For stronger type checks, it is better not to specify the response model if it matches the return type.
```suggestion
@frame_router.get("/bounds")
```